### ISSUE TITLE
[Feature request] Option to disable the default output entirely

### ISSUE THREAD

**Issue by gmolau** [sharkdp/hyperfine#193]
I think it would be nice to be able to disable the standard output (with the progress bar) entirely. This would make it possible to cleanly pipe the JSON output to another process (via `--export-json /dev/stdout`). It seems like making the `--style` option accept `none` would be a good way to implement this. I'm unfortunately not familiar with Rust (otherwise I would just submit a PR), but maybe this could be considered for the future.

**Comment by sharkdp** [comment]
Great idea, thank you very much.

In the meantime, you can use process substitution to pipe the output to another process. Not as beautiful as it could be, I agree:
```bash
> hyperfine 'sleep 0.1' --export-json >(jq '.results[0].mean') >/dev/null
0.10603645827285715
```

**Comment by knidarkness** [comment]
Hi @sharkdp, I am currently using hyperfine and find it quite nice tool. If this issue is still relevant I could implement it. Is it?

**Comment by sharkdp** [comment]
It is, yes. 

That would be great

**Comment by knidarkness** [comment]
Great, could you assign it to me and I`ll check it on the weekend?

**Comment by sharkdp** [comment]
closed via #224 by @knidarkness

**Comment by sharkdp** [comment]
This works now:
```bash
hyperfine --style=none --export-json /dev/stdout 'sleep 0.1' 'sleep 0.2' | jq '.results[].mean'
```
